app/models/remote_execution_feature.rb

```diff
@@ -0,0 +1,36 @@
+class RemoteExecutionFeature < ActiveRecord::Base
+  attr_accessible :label, :name, :provided_input_names, :description, :job_template_id
+
+  validate :label, :name, :presence => true, :unique => true
+
+  belongs_to :job_template
+
+  extend FriendlyId
+  friendly_id :label
+
+  def provided_input_names
+    self.provided_inputs.to_s.split(',').map(&:chomp)
+  end
+
+  def provided_input_names=(values)
+    self.provided_inputs = Array(values).join(',')
+  end
+
+  class << self
```

Comment:
is there some reason (autoloader maybe) for this syntax instead of `def self.feature(label)` which I'd prefer?